### ISSUE TITLE
How to train!!!

### ISSUE THREAD

**Issue by Yaoyaolingbro** [ProteinDesignLab/protpardelle#9]
The data url you provided is not match with your code need. What should I do? I've extract the eval/test/train_pdb_keys.list by the `chain_set_split.json`. But when I tar the *pdb.gz, it doesn't match with the key in chain_set_split.

Thanks for your details reply!

**Comment by alexechu** [comment]
Yes, there are a few PDB IDs that have been obsoleted since the splits were defined (they are still split the same way, but the IDs have changed.) I'm looking into if there an easier way to set this up out of the box for users, and will also add the training code soon.

**Comment by Yaoyaolingbro** [comment]
Thanks for your contribution to protein generative research!
